tidy3d/components/device/source.py

```diff
@@ -30,20 +30,48 @@ def plot_params(self) -> PlotParams:
         return plot_params_heat_source
 
 
-class UniformHeatSource(HeatSource):
+class HeatSource(DeviceSource):
     """Volumetric heat source.
 
     Example
     -------
-    >>> heat_source = UniformHeatSource(rate=1, structures=["box"])
+    >>> heat_source = HeatSource(rate=1, structures=["box"])
+
+    An analytical expression can also be defined
+    --------------------------------------------
```

Comment:
Typically we have sections Note and Example in docstrings

Reply:
How would I go about this? Do I do something like this?

Reply:
yeah, similar to, for example https://github.com/flexcompute/tidy3d/blob/develop/tidy3d/components/monitor.py#L391-L425